lldb/include/lldb/Symbol/UnwindPlan.h

```diff
@@ -192,6 +202,8 @@ class UnwindPlan {
           const uint8_t *opcodes;
           uint16_t length;
         } expr;
+        // For m_type == isConstant
```

Comment:
Are you planning to upstream these changes using a unit test to llvm.org?
Otherwise this should be wrapped in // BEGIN SWIFT comment blocks.

Reply:
It's already upstreamed.
I talked to @jasonmolenda about this and unfortunately there is no good way of unit testing it